editor/components/inner-blocks/index.js

```diff
@@ -43,15 +43,19 @@ class InnerBlocks extends Component {
 	updateNestedSettings() {
 		const {
 			blockListSettings,
-			allowedBlocks: nextAllowedBlocks,
+			allowedBlocks,
+			lock,
```

Comment:
Mentioned at https://github.com/WordPress/gutenberg/pull/6991#discussion_r196916721: If we call this `templateLock` at the editor level, why do we not call it the same here?

Reply:
My idea was to try to move locking out of being a template concept. I wanted the lock to be a setting blocks can make use. E.g. have settings that enable or disable locking.
But yes given that now it's main use case is templates I updated the name to be templateLock.